packages/math/src/bn.test.ts

```diff
@@ -475,6 +475,7 @@ describe('Math - BN', () => {
     expect(bn.parseUnits('100,100,100.00002', 5).toHex()).toEqual(bn('10010010000002').toHex());
     expect(bn.parseUnits('.').toHex()).toEqual(bn('0').toHex());
     expect(bn.parseUnits('.', 5).toHex()).toEqual(bn('0').toHex());
+    expect(bn.parseUnits('1', 0).toHex()).toEqual(bn('1').toHex());
```

Comment:
Should we try more values other than `1`?

Reply:
Added a couple more test cases